### ISSUE TITLE
The dimension of the CLIP-ViT-B-16-views.hdf5

### ISSUE THREAD

**Issue by honghd16** [CrystalSixone/VLN-GOAT#3]
Hi, thanks for your great work!

I am running it all good but just with a little question.
Since the visual feature is extracted by CLIP-B/16, I am just wondering why the feature dimension is set to 768 instead of 512, which is also reflected in the CLIP-ViT-B-16-views.hdf5 with 1768 for each viewpoint. 
https://github.com/CrystalSixone/VLN-GOAT/blob/290e70b3f099cb4d120b3b66a969aa0f4e7a7b4a/pretrain_src/config/r2r_GOAT_model_config.json#L8

Thank you!

**Comment by CrystalSixone** [comment]
Hi, thank you for your attention! 
The feature dimension is 768 because we use the layer output before class mapping in the CLIP model. In the CLIP-ViT-B-16-views.hdf5 file, the additional 1000 dimensions correspond to image classification logits.

**Comment by honghd16** [comment]
Thank you so much! Could I know whether the chosen layer before class mapping is based on experimental results since other works mostly use the 512 dimension like scaleVLN and EnvEdit.

**Comment by CrystalSixone** [comment]
The dimension in EnvEdit is also 768 (we use the feature file from EnvEdit). Based on our experiments, there is not much difference between using 768 and 512 dimensions.

**Comment by honghd16** [comment]
Thank you!
